tests/test_prompt.py

```diff
@@ -35,10 +36,11 @@ def test_convert_to_str(mocker, raw_var, rendered_var):
     result = prompt.render_variable(env, raw_var, context)
     assert result == rendered_var
 
-    # Make sure that non str variables are conerted beforehand
-    if not isinstance(raw_var, basestring):
-        raw_var = str(raw_var)
-    from_string.assert_called_once_with(raw_var)
+    # Make sure that non None non str variables are conerted beforehand
+    if raw_var is not None:
```

Comment:
Right, because the assertion doesn't apply anymore (there's still the assertion above checking the return value).

I could assert that it's not called I suppose..